libcontainer/rootfs_linux.go

```diff
@@ -107,12 +121,12 @@ func mountToRootfs(m *configs.Mount, rootfs, mountLabel string) error {
 		if err := os.MkdirAll(dest, 0755); err != nil {
 			return err
 		}
-		return syscall.Mount(m.Source, dest, m.Device, uintptr(m.Flags), "")
+		return mountPropagate(m.Source, dest, m.Device, m.Flags, "", m.PropagationFlags)
```

Comment:
So, I'm not sure now that `mountLabel` belongs to libcontainer at all.
But I still want to have `mount.Mount` method, maybe it can accept rootfs and mountlabel for now. Because it's sorta tiring to write all this struct fields each time.